### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by skykatik**
Pattern matching, lambda instanceof, **whitespaces**, and some typos

### CODE REVIEW

File: core/src/mindustry/maps/SectorDamage.java [hunk 1]
```diff
@@ -291,7 +291,7 @@ public static void writeParameters(SectorInfo info){
 
                 sumHealth += unit.health*healthMult + unit.shield;
                 sumDps += unit.type.dpsEstimate;
-                if(unit.abilities.find(a -> a instanceof RepairFieldAbility) instanceof RepairFieldAbility h){
+                if(unit.abilities.find(RepairFieldAbility.class::isInstance) instanceof RepairFieldAbility h){
```
Comment (summetdev) [inline core/src/mindustry/maps/SectorDamage.java:294]: Maybe leave the ʻinstanceof` keyword here?

File: core/src/mindustry/type/Item.java [hunk 2]
```diff
@@ -53,6 +53,6 @@ public ContentType getContentType(){
 
     /** Allocates a new array containing all items that generate ores. */
     public static Seq<Item> getAllOres(){
-        return content.blocks().select(b -> b instanceof OreBlock).map(b -> ((Floor)b).itemDrop);
+        return content.blocks().select(OreBlock.class::isInstance).map(b -> ((Floor)b).itemDrop);
```
Comment (summetdev) [inline core/src/mindustry/type/Item.java:56]: Same ^

**Review by summetdev — APPROVED** [review]

File: core/src/mindustry/type/Item.java (hunk 2 above)
Comment (skykatik) [reply core/src/mindustry/type/Item.java:56]: Maybe yes

**Comment by skykatik** [comment]
.-.

File: core/src/mindustry/logic/LogicOp.java [hunk 3]
```diff
@@ -4,7 +4,7 @@
 import arc.util.*;
 
 public enum LogicOp{
-    add("+", (a, b) -> a + b),
+    add("+", Double::sum),
```
Comment (Anuken) [inline core/src/mindustry/logic/LogicOp.java:7]: not compatible with android

File: core/src/mindustry/logic/LogicOp.java [hunk 4]
```diff
@@ -29,7 +29,7 @@
     noise("noise", LExecutor.noise::rawNoise2D),
 
     not("not", a -> ~(long)(a)),
-    abs("abs", a -> Math.abs(a)),
+    abs("abs", Math::abs),
```
Comment (Anuken) [inline core/src/mindustry/logic/LogicOp.java:32]: fails to compile for some java versions

File: core/src/mindustry/graphics/MenuRenderer.java [hunk 5]
```diff
@@ -42,7 +42,7 @@ public MenuRenderer(){
     private void generate(){
         world.beginMapLoad();
         Tiles tiles = world.resize(width, height);
-        Seq<Block> ores = content.blocks().select(b -> b instanceof OreBlock);
+        Seq<Block> ores = content.blocks().select(OreBlock.class::isInstance);
```
Comment (Anuken) [inline core/src/mindustry/graphics/MenuRenderer.java:45]: this is awful, never use reflection in a place where standard java works just fine

File: core/src/mindustry/core/Control.java [hunk 6]
```diff
@@ -185,7 +186,7 @@ public Control(){
                 if(state.isCampaign()){
                     ui.announce("[accent]" + state.rules.sector.name() + "\n" +
                         (state.rules.sector.info.resources.any() ? "[lightgray]" + bundle.get("sectors.resources") + "[white] " +
-                            state.rules.sector.info.resources.toString(" ", u -> u.emoji()) : ""), 5);
+                            state.rules.sector.info.resources.toString(" ", UnlockableContent::emoji) : ""), 5);
```
Comment (Anuken) [inline core/src/mindustry/core/Control.java:189]: there's no reason to use method references when the result is longer code

**Review by Anuken — CHANGES_REQUESTED** [review]

**Pull request merged**